packages/salesforcedx-utils-vscode/src/commands/notificationService.ts

```diff
@@ -125,9 +121,7 @@ export class NotificationService {
         showButtonText,
         showOnlyStatusBarButtonText
       );
-      if (selection && selection === showButtonText) {
```

Comment:
Do we intend to remove this? show button won't have an effect then

Reply:
Yes, I'm working on another PR which will modify some of the notification service code and I'll address the show button there. Just wanted to remove the channel service dependency for now.